powerstore/resource_protection_policy_test.go

```diff
@@ -25,8 +25,8 @@ func TestAccProtectionPolicy_Create(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("powerstore_protectionpolicy.test", "name", "protectionpolicy_acc_new"),
 					resource.TestCheckResourceAttr("powerstore_protectionpolicy.test", "description", "Test CreateProtectionPolicy"),
-					resource.TestCheckResourceAttr("powerstore_protectionpolicy.test", "replication_rule_ids.0", "5d45b173-9a85-473e-8ab8-e107f8b8085e"),
-					resource.TestCheckResourceAttr("powerstore_protectionpolicy.test", "snapshot_rule_ids.0", "4be81573-c0e6-4956-a32f-a0e396a9b86d"),
+					resource.TestCheckResourceAttr("powerstore_protectionpolicy.test", "replication_rule_ids.0", replicationRuleID),
+					resource.TestCheckResourceAttr("powerstore_protectionpolicy.test", "snapshot_rule_ids.0", snapshotRuleID),
 				),
 			},
```

Comment:
Can you point me what exactly you've done to achieve this **_"Modified protectionPolicy reference variable to variable as per latest changes on gopowerstore side."_** (stated in the description)